### PR TITLE
Use run-all for tf modules

### LINKED ISSUE #23: Running deploy-serve / transform before core doesn't automatically apply core first
Terragrunt commands need to be updated to use run-all in the sub-modules

### PR DESCRIPTION
**Pull request by jjgriff93**
Fixes #23 

Adds `run-all` commands to all deploy make commands

### CODE REVIEW

**Review by t-young31 — APPROVED** [review]
🚀

**Review by t-young31 — CHANGES_REQUESTED** [review]
just tried it from a blank slate and [doesn't seem like it works](https://github.com/t-young31/FlowEHR/actions/runs/3995500453/jobs/6854456930#step:5:1759) 😢

File: Makefile [hunk 1]
```diff
@@ -56,17 +56,17 @@ deploy: az-login ## Deploy all infrastructure
 deploy-core: az-login ## Deploy core infrastructure
 	$(call target_title, "Deploy Core Infrastructure") \
 	&& . ${MAKEFILE_DIR}/scripts/load_env.sh \
-	&& terragrunt apply ${MAKEFILE_DIR}/infrastructure/core
+	&& terragrunt run-all apply --terragrunt-working-dir ${MAKEFILE_DIR}/infrastructure/core --terragrunt-non-interactive
```
Comment (t-young31) [inline Makefile:59]:
```suggestion
		&& cd ${MAKEFILE_DIR}/infrastructure/serve \
	&& terragrunt run-all apply --terragrunt-include-external-dependencies --terragrunt-non-interactive
```
and likewise for the others works ok

Comment (jjgriff93) [reply Makefile]: Works like a charm, thanks!

**Review by t-young31 — APPROVED** [review]

**Pull request merged**